ios/ReactNativeCameraKit/Types.swift

```diff
@@ -105,6 +105,19 @@ public enum ZoomMode: Int, CustomStringConvertible {
     }
 }
 
+@objc(CKResizeMode)
+public enum ResizeMode: Int {
```

Comment:
Can we make it consistent with the other ones, like the one above?

No `= 0`, implements `CustomStringConvertible`, `description` instead of `stringValue`

Reply:
@lichstam what about this one?